README.md

```diff
@@ -74,48 +74,52 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Install flox
-      uses: flox/install-flox-action@v1.0.0
+      uses: flox/install-flox-action@v1
 
     - name: Build
       run: flox build
 ```
 
-### Using substituters for caching
+### 🚀 Add caching for faster CI
 
-You can have this action configure the substitutes for you. This will allow you to push build artifacts to a remote Nix store, and have subsequent builds substitute paths using that same store.
+Caching with flox can be achieved by configuring what is known as
+`substituter`.
```

Comment:
Curious: what is being substituted here, and is it something that should be explained? If not (taking the other extreme 😄) is it worth changing `substituter` and `substituter_key` so users don't encounter it?

Reply:
Personally I'd be more happy if we would use the work `cache` and `cache_key` here, since we are configuring the Nix's substitutors in the `install-flox-action` with a purpose of providing a binary cache.

I would be more then happy to to give some UX polish after this PR.